homeassistant/components/cover/__init__.py

```diff
@@ -481,7 +481,7 @@ async def async_toggle_tilt(self, **kwargs: Any) -> None:
     def _get_toggle_function(
         self, fns: dict[str, Callable[_P, _R]]
     ) -> Callable[_P, _R]:
-        if CoverEntityFeature.STOP | self.supported_features and (
+        if CoverEntityFeature.STOP in self.supported_features and (
```

Comment:
If we want to use `in` for the check instead of the bitwise operator `&` we need to implement `supported_features_compat`, and use that instead.

Example:

https://github.com/home-assistant/core/blob/c96f9864c50744b08e0190d9dd83b0a422952f94/homeassistant/components/climate/__init__.py#L665

Reply:
Let's then just use `&` here for now. Changing `CoverEntityFeature` to support `in` can be done in a follow-up.